content/en/hosting/costs/_index.md

```diff
@@ -0,0 +1,95 @@
+
+By looking at dozens of instances and focusing on the highest usage ones, an accurate representation can be derived of a higher cost a self hosting partner.  
+
+Here's some real costs for two week period for two instances that have been renamed to simply **A** and **B**:
+
+| Per Month | A | B | 
+| --- | --- | --- | --- |
+| $/User | $0.05 | $0.10 |  |
+| $ | $94 | $68 |  |
+| Active Users (30 days) | 2000 | 661 |  |
+| DB Size (Millions) | 7.5M | 10.7M |  |
+| DB Growth (Thousands) | 123.2K | 248.0K |  |
+| RAM | $16 | $22 |  |
+| CPU | $6 | $8 |  |
+| Storage | $53 | $24 |  |
+| CPU max used (vCPUs) | 2.9 | 8 |  |
+| RAM max (Gigs) | 20 | 15 |  |
+| Storage Max PVC (TBs) | 0.96 | 0.42 |  |
+
+Deployment **A** has a lot of active users (`2000/mo`), but a slower database growth (`123.2K/mo`) as compared to deployment **B** which has fewer users (`661/mo`) but faster growth (`248K/mo`). These are the types of variations that can cause changes in costs mentioned in the next section.
+
+### Accuracy
+
+As our research was based on live instances, the costs are realistic.  However, since not all users will use the CHT in the exact same way and not all deployments have the same number of users and work flows, these numbers are still subject to some per-deployment fluctuation.
+
+Further, the costs in the table above only cover the variable costs.  That is, the costs incurred of actual usage with in an EKS cluster.  In a multi-tenant deployment where you have maybe three instances (Staging, User Acceptance Testing and Production), this can be critical to know what the three different deployments incurred costs are within your EKS cluster.  
+
+Importantly, all production CHT 4.x deployments have a fixed cost to run the EKS cluster or generic Kubernetes if you're not on AWS. This is because all clusters contain servers that are always on and fully available to you. This cost is not yet included in this document and Medic hopes to include this cost over time.
+
+###  Production vs Development
+
+A production deployment means the CHT is hosted either at a cloud provider or in a datacenter. Both provide high quality connectivity, power and cooling so the cost is higher than a development instance.  This ensures users do not have service interruptions and can continue to have trust the CHT will be up when they need to deliver care.
+
+When analyzing the hosting total cost of ownership (TCO), only production instances were looked at.
+
+It should be noted that a development environment can be no cost (or very low cost).  Assuming a developer already has a laptop, this is all that is needed to host a development instance. 
+
+Running an easy to set up Docker based development environment (instead of Kubernetes) on a laptop is an acceptable practice: there is no expectation of high uptime for development instances.
```

Comment:
Consider linking to the app dev hosting docs.

Reply:
yup yup!  fixed in 9cd2749a